functions/fit/p_bestfitcylinderfrompoints.cpp

```diff
@@ -102,3 +87,74 @@ bool BestFitCylinderFromPoints::setUpResult(Cylinder &cylinder){
 
     return bestFitCylinder(this, cylinder, points, usablePoints);
 }
+
```

Comment:
warum muss der gleiche Code zweimal implementiert werden? (P_bestfitCylinder.cpp)

Reply:
die Methoden unterscheiden sich in den Datenquellen, die verwendet werden Observation vs.  Point